### PR TITLE
various ui cleanups.

### PR DESCRIPTION
**Pull request by SteveMaier-IRT**
[//]: # (🚨 Please review the CONTRIBUTING.md in this repository. 💔Thank you!)

### Issue
*Link your PR to an issue*

Fixes #___

### Description
*Please describe your pull request.*

### Testing
*Describe the best way to test or validate your PR.*

### CODE REVIEW

File: src/electionguard-ui/ElectionGuard.UI/Resources/Styles/EgStyles.xaml [hunk 1]
```diff
@@ -235,6 +235,13 @@
                         Source="world.png"
                         ToolTipProperties.Text="{helpers:Translate SwitchLanguages}" />
                 </HorizontalStackLayout>
+                <Label
+                    FontAttributes="Bold"
+                    FontSize="Medium"
+                    HorizontalOptions="CenterAndExpand"
+                    LineBreakMode="TailTruncation"
+                    Text="{Binding ErrorMessage}"
+                    TextColor="Red" />
```
Comment (SteveMaier-IRT) [inline src/electionguard-ui/ElectionGuard.UI/Resources/Styles/EgStyles.xaml:244]: Added error messaging in the template

File: src/electionguard-ui/ElectionGuard.UI/Services/TallyManager.cs [hunk 2]
```diff
@@ -378,12 +378,19 @@ private async Task<IEnumerable<CiphertextBallot>> GetBallotsByState(string elect
...
+                    userId,
+                    keyCeremony,
+                    publicKeys,
+                    backups.ToDictionary(k => k.GuardianId!, v => v.Backup!)) ?? throw new ElectionGuardException(nameof(userId));
+                return guardian;
+            }
+            catch (Exception ex)
+            {
+                throw new ElectionGuardException(AppResources.ErrorLoadingGuardian, ex);
+            }
```
Comment (SteveMaier-IRT) [inline src/electionguard-ui/ElectionGuard.UI/Services/TallyManager.cs:393]: Handling path not found

File: src/electionguard-ui/ElectionGuard.UI/Views/LoginPage.xaml [hunk 3]
```diff
@@ -15,15 +15,11 @@
         <Entry
             windows:VisualElement.AccessKey="N"
             Placeholder="{helper:Translate LoginPlaceholder}"
+            ReturnCommand="{Binding LoginCommand}"
```
Comment (SteveMaier-IRT) [inline src/electionguard-ui/ElectionGuard.UI/Views/LoginPage.xaml:18]: handles typing name and hitting enter on keyboard

File: src/electionguard-ui/ElectionGuard.UI/ViewModels/TallyProcessViewModel.cs [hunk 4]
```diff
@@ -221,12 +221,12 @@ private bool CanJoinTally()
     {
         return Tally?.State == TallyState.PendingGuardiansJoin &&
             !CurrentUserJoinedAlready() &&
-            !AuthenticationService.IsAdmin;
+            !IsAdmin;
```
Comment (john-s-morgan) [inline src/electionguard-ui/ElectionGuard.UI/ViewModels/TallyProcessViewModel.cs:224]: This was a lot of why the Auth service was pulled into this VM. Can we remove it?

File: src/electionguard-ui/ElectionGuard.UI/Views/LoginPage.xaml (hunk 3 above)
Comment (john-s-morgan) [reply src/electionguard-ui/ElectionGuard.UI/Views/LoginPage.xaml:18]: 🙌🏼

File: src/electionguard-ui/ElectionGuard.UI/Views/LoginPage.xaml.cs [hunk 5]
```diff
@@ -5,5 +5,12 @@ public partial class LoginPage
     public LoginPage(LoginViewModel vm) : base(vm)
     {
         InitializeComponent();
+        Loaded += LoginPage_Loaded;
+    }
+
+    private void LoginPage_Loaded(object? sender, EventArgs e)
+    {
+        var vm = BindingContext as LoginViewModel;
+        vm.OpenSettingsUnsetData();
```
Comment (john-s-morgan) [inline src/electionguard-ui/ElectionGuard.UI/Views/LoginPage.xaml.cs:14]:
```suggestion
        vm?.OpenSettingsUnsetData();

```

**Review by john-s-morgan — APPROVED** [review]

**Pull request merged**